pkg/snapshot/store.go

```diff
@@ -1596,3 +1595,8 @@ func WaitForDefaultBslAvailableAndSynced(ctx context.Context, veleroNamespace st
 		}
 	}
 }
+
+func isMinioMigration(clientset kubernetes.Interface, namespace string) bool {
+	_, err := clientset.CoreV1().ConfigMaps(namespace).Get(context.TODO(), SnapshotMigrationArtifactName, metav1.GetOptions{})
+	return err == nil
```

Comment:
It depends on what behavior we want if other errors occur.  If it fails for a different reason, then we assume it's not a Minio migration.  That may or may not actually be the case, but seems like defaulting to false in those cases is safer since the prefix only gets added when true?  Maybe I can add a check for the `IsNotFound` error and log a message if the error is something else, but still default to returning false in those cases?

Reply:
Just pushed the above change.